### PR TITLE
feat(prejoin): Add name from jwt to prejoin screen

### PR DESCRIPTION
**Pull request by vp8x8**
* The prejoin display name is pre-filled with the name from the jwt.

### CODE REVIEW

File: react/features/base/settings/middleware.js [hunk 1]
```diff
@@ -115,6 +122,25 @@ function _maybeSetAudioOnly(
...
+ * Updates the display name to the one in JWT for vpaas meetings.
+ *
+ * @param {Store} store - The redux store.
+ * @private
+ * @returns {void}
+ */
+function _maybeUpdateDisplayName({ dispatch, getState }) {
+    const state = getState();
+
+    if (isVpaasMeeting(state)) {
```
Comment (damencho) [inline react/features/base/settings/middleware.js:135]: Is there a reason to limit this only for vpass? Can we use the jwt name in general if it exist. Can we make it more generic, is what I mean?

Comment (vp8x8) [reply react/features/base/settings/middleware.js:135]: Good point! I'll check the shape of the jwt and make it more generic.

Comment (damencho) [reply react/features/base/settings/middleware.js:135]: Yep name is in the general jwt structure as described here: https://github.com/jitsi/lib-jitsi-meet/blob/master/doc/tokens.md#payload

Comment (vp8x8) [reply react/features/base/settings/middleware.js]: Done.

**Review by damencho — APPROVED** [review]

**Pull request merged**